### PR TITLE
Reverse continue fix and updates for TTD info tracking.

### PR DESCRIPTION
**Pull request by mrkmarron**
- [x] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [x] documentation is changed or added

##### Affected core subsystem(s)

chakrashim, TTD

##### Description of change
Fix for reverse-continue.
Update tracking for TTD.

PR-URL: https://github.com/nodejs/node-chakracore/pull/231/commits/318c0edf3e6bf795aa7cc6339842302d8dc15bcf

### CODE REVIEW

**Comment by mrkmarron** [comment]
This depends on updating core to latest release/2.0.

**Review by kfarnung — APPROVED** [review]
LGTM, I can give these changes a try against inspector today.

**Comment by kfarnung** [comment]
Created PR #232 to integrate the release/2.0 changes.

File: src/node_os.cc [hunk 1]
```diff
@@ -176,6 +176,13 @@ static void GetCPUInfo(const FunctionCallbackInfo<Value>& args) {
     fields[field_idx++] = ci->cpu_times.irq;
     model_argv[model_idx++] = OneByteString(env->isolate(), ci->model);
 
+#if ENABLE_TTD_NODE
+    if(s_doTTRecord || s_doTTReplay) {
```
Comment (kfarnung) [inline src/node_os.cc]: Can you fix this lint issue?

src\node_os.cc:180:  Missing space before ( in if(  [whitespace/parens] [5]

File: src/node_os.cc [hunk 2]
```diff
@@ -223,6 +230,12 @@ static void GetLoadAvg(const FunctionCallbackInfo<Value>& args) {
   Local<ArrayBuffer> ab = array->Buffer();
   double* loadavg = static_cast<double*>(ab->GetContents().Data());
   uv_loadavg(loadavg);
+
+#if ENABLE_TTD_NODE
+  if(s_doTTRecord || s_doTTReplay) {
```
Comment (kfarnung) [inline src/node_os.cc]: here too

src\node_os.cc:235:  Missing space before ( in if(  [whitespace/parens] [5]

**Review by kfarnung — APPROVED** [review]
Other than the two cpplint issues, it LGTM.

**Comment by kfarnung** [comment]
Pushed a patch and kicked off a CI:

https://ci.nodejs.org/job/chakracore-test-pull-request/20/

**Pull request merged**